doc/source/whatsnew/v0.24.0.txt

```diff
@@ -74,6 +74,7 @@ Other Enhancements
 - :func:`Series.mode` and :func:`DataFrame.mode` now support the ``dropna`` parameter which can be used to specify whether NaN/NaT values should be considered (:issue:`17534`)
 - :func:`to_csv` now supports ``compression`` keyword when a file handle is passed. (:issue:`21227`)
 - :meth:`Index.droplevel` is now implemented also for flat indexes, for compatibility with :class:`MultiIndex` (:issue:`21115`)
+- :meth:`Series.droplevel` and `DataFrame.droplevel` are now implemented (:issue:`20342`)
```

Comment:
need ref for DataFrame.droplevel

Reply:
could you please explain more, not sure what you mean

Reply:
You have

    `DataFrame.droplevel`

Should be

    :meth:`DataFrame.droplevel`

Reply:
thank you!